scapy/sendrecv.py

```diff
@@ -286,13 +286,13 @@ def sendp(x, inter=0, loop=0, iface=None, iface_hint=None, count=None, verbose=N
                       verbose=verbose, realtime=realtime, return_packets=return_packets)
 
 @conf.commands.register
-def sendpfast(x, pps=None, mbps=None, realtime=None, loop=0, file_cache=False, iface=None):
+def sendpfast(x, pps=None, mbps=None, realtime=None, loop=0, preload_pcap=False, iface=None):
```

Comment:
Please keep the same API. This is not supposed to be a `tcpreplay` option, but a Scapy function parameter.